package.json

```diff
@@ -9,13 +9,12 @@
     "check:eslint": "tsc -P eslint-rules/",
     "test:eslint": "jest eslint-rules/tests/*.test.js",
     "test:unit": "cross-env ELECTRON_RUN_AS_NODE=1 NODE_OPTIONS='--max_old_space_size=4096' ./node_modules/.bin/electron ./node_modules/jest/bin/jest --detectOpenHandles --silent --testLocationInResults --config ./app/jest.unit.config.js",
-    "test:unit:cov": "yarn test:unit --coverage",
+    "test:unit:cov": "yarn test:unit",
     "test:script": "jest --silent --config ./script/jest.config.js",
-    "test:script:cov": "yarn test:script --coverage",
+    "test:script:cov": "yarn test:script",
```

Comment:
Can we also remove these `:cov` scripts entirely?

Reply:
Absolutely, I thought I had already!